packages/cli-kit/src/public/node/hooks/prerun.ts

```diff
@@ -81,3 +86,18 @@ function findAlias(aliases: string[]) {
     return existingAlias.replace(/:/g, ' ')
   }
 }
+
+function warnOnAvailableUpgrade() {
+  const cliDependency = '@shopify/cli'
+  const currentVersion = CLI_KIT_VERSION
```

Comment:
What about skipping this for nightlies and snapshots? It's probably not useful in those cases and the version comparison won't probably work as expected.

Reply:
What about adding a comment? It might be confusing if you don't know that nightlies start with `0.0.0`